src/repositories/qr.repository.ts

```
@@ -0,0 +1,36 @@
+import { Pool } from 'pg';
+import logger from '@/configs/logger.config';
+import { DBError } from '@/exception';
+import { QRLoginToken } from '@/types/models/QRLoginToken.type';
+
+export class QRLoginTokenRepository {
+    constructor(private pool: Pool) { }
+
+    async createQRLoginToken(token: String, userId: number, ip: string, userAgent: string): Promise<void> {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)